---
geometry_msgs/Point initial_translation
bool success
